app/models/contact.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: contacts
+#
+#  id               :bigint(8)        not null, primary key
+#  first_name       :string(255)
+#  last_name        :string(255)
+#  phone            :string(255)
+#  fax              :string(255)
+#  email            :string(255)
+#  contactable_type :string(255)
+#  contactable_id   :bigint(8)
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#
+
+class Contact < ApplicationRecord
+    belongs_to :contactable, polymorphic: true
+    has_one :web_url, as: :web_urlable
```

Comment:
in einer anderen datei sind die reihenfolgen unterschiedlich:

```
has_one :geo_location, as: :geo_locateable
belongs_to :adressable, polymorphic: true
```

da gibt es doch best practice, in welcher reihenfolge das geschehen sollte, oder?!

Reply:
Ich glaube das ist relativ egal. An solchen Dingen sollte man sich nicht aufhalten, wenn der main app server bis  Mitte Mai fertig sein soll.

Reply:
welchen stackoverflow post?